### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by tamarazuk**
With the release of v0.1.0 of UI Kit and SDK, we'll need update the template

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ function App() {
   }, []);
 
   function onClick() {
-    HelpScout.showNoty("success", "Hello from the sidebar app");
+    HelpScout.showNotification("SUCCESS", "Hello from the sidebar app");
```
Comment (deivamagalhaes) [inline src/App.tsx]: Is there a constant we can use here?

Comment (tamarazuk) [reply src/App.tsx]: Great call! Fixed in f3745452d3e6f06ffa83ea38de98d12f6eae92c3

**Review by deivamagalhaes — APPROVED** [review]
👍

File: package.json [hunk 2]
```diff
@@ -9,8 +9,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@helpscout/javascript-sdk": "^0.0.11",
-    "@helpscout/ui-kit": "^0.0.37",
+    "@helpscout/javascript-sdk": "^0.1.0",
+    "@helpscout/ui-kit": "^0.1.0",
+    "@vitejs/plugin-basic-ssl": "^1.0.1",
```
Comment (Charca) [inline package.json]: Should this be a `devDependency`?

**Review by Charca — APPROVED** [review]
I left a minor comment but PR looks good to me!

File: package.json (hunk 2 above)
Comment (tamarazuk) [reply package.json]: ah yes, thanks!

Comment (tamarazuk) [reply package.json]: fixed in 56c92b92ca79977d44e51176cf54a77635bdc75f

**Pull request merged**